build/crd/percona/generated/pgv2.percona.com_perconapgbackups.yaml

```diff
@@ -132,7 +132,10 @@ spec:
                     - bucket
                     type: object
                   name:
-                    description: The name of the repository
+                    default: repo1
+                    description: |-
+                      The name of the repository
+                      K8SPG-511. The default value should be added to this field to avoid the problem under openshift installation
```

Comment:
Do we need to have task number in the desc? When you add a comment on a field kubebuilder treats as a field description. Did you intended to have it on CRD desc?